migrate/20231122_spdk_upgrade.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:spdk_installation) do
+      column :id, :uuid, primary_key: true
+      column :version, :text, null: false
+      column :allocation_weight, :Integer, null: false
+      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      foreign_key :vm_host_id, :vm_host, type: :uuid
+      unique [:vm_host_id, :version]
+    end
+
+    alter_table(:vm_storage_volume) do
+      add_foreign_key :spdk_installation_id, :spdk_installation, type: :uuid, null: true
+    end
+  end
+end
```

Comment:
You can INSERT INTO spdk_installations table as part of the migration for existing hosts (and UPDATE vm_storage_volume), which would remove necessity to specifically do things in prod for migration. It would also help for not breaking development environments, but that is a secondary concern.

Though I'm not sure if we need to do anything in data plane before updating the control plane. If so, it would be indeed better to add spdk_installations as a separate migration step.